### ISSUE TITLE
Many blocks do not update/work properly when used as dome blocks

### ISSUE THREAD

**Issue by TheLoneWolfling** [BrainSlugs83/NewBiospheresMod#20]
The most obvious instances of this are water / lava, but there are others as well. For instance, piston dome blocks do not extend / retract, things aren't put into dispenser dome blocks properly, etc.

**Comment by BrainSlugs83** [comment]
Yeah, I figured that would be the case for a lot of blocks -- honestly, I wonder at the value of having a piston or a liquid as the dome block -- tnt seems to work -- I do wonder about chests (that could certainly be interesting...).

It's not a "wontfix", but unless it's really holding someone back, I'm probably going to put it as a low priority for right now (especially considering the crash bugs, etc.).

**Comment by nanakisan** [comment]
If I may hijack this issue instead of posting a new one. It would seem your config options for the mod completely disagree with the use of modded blocks. Even when using a direct mod-id of the block I want for the dome. It simple defaults it back to glass. Either i'm doing something wrong or the mod simply loves glass too much.
